docs/1-Applying/how_to_apply.md

```diff
@@ -40,4 +44,6 @@ If a friend can refer to you a company (maybe a company they've worked) this can
 
 However - I tend to not ask for referals or give them because I don't work technically with many of my friends. I can vouch for them being kind and thoughtful and hard working (and same for them about me) but neither of us really know how well the other works on technical things and so unless they're a close friend I'd probably feel weird referring them.
 
-What I *have* done (and asked for) is introduced my friends to recruiters, in sort of a "here's my friend they're pretty cool and also interested in your company" and less of a "I recommend this person to work here" kind of way. Maybe this is what most referals are anyways, but there you go.
+What I *have* done (and asked for) is introduced my friends to recruiters, in sort of a "here's my friend they're pretty cool and also interested in your company" and less of a "I recommend this person to work here" kind of way. Even if the most your friend can say about you is that you're a nice person, this is still valuable. Very broadly speaking, most companies look for two things: "this person is technically competent" and "this person will work effectively in a team" (i.e. a fancy way of saying "not an asshole"). A non-technical referral is still useful for the recruiter since they don't need to worry about the latter as much.
+
+Of course, for a friend to introduce you as cool person, this requires you to be a cool person. This is a reputation you build throughout your time as a student. Be nice to everyone you meet, not just the ones who can do you favors! You never know who might end up in a position to give referrals at a desirable company.
```

Comment:
Hmm interesting. I had something in mind when writing this but I need to think more about what message I want to say, and if it's general enough to be worth including.

What I had in mind was: in competitive environment, there's a tendency for a non-trivial subset of students to want to put people in a pecking order (can't deny I've been tempted a few times...). Since the previous paragraph talked about referrals, I'm worried some people will conclude that they should select their friends careful based on who has the potential to provide referrals in the future. However I want to say that that pecking order is mostly illusory. So the best thing to do is to assume the best of everybody you meet (which, still following the previous paragraph, means more specifically "assume they'll do well eventually and end up in a good place"). Thoughts?